frontend/src/components/funding/FundingAgreementsTab.vue

```diff
@@ -59,12 +61,42 @@ export default {
     ...mapState(useAuthStore, ['userInfo']),
   },
 
-  created() {
+  async created() {
+    this.loading = true
     this.format = format
     this.FUNDING_AGREEMENT_STATUS_CODES = FUNDING_AGREEMENT_STATUS_CODES
+    await this.loadApprovedSuppApps()
+    this.loading = false
   },
 
   methods: {
+    async loadApprovedSuppApps() {
+      try {
+        this.loading = true
+        const applications = await ApplicationService.getActiveApplications()
```

Comment:
the Funding tab can display all Historical Funding Agreement (expired). Should we also display historical supplementary funding?

Reply:
I don't think we have the functionality to support expired yet - I will write a note though for it to be picked up in future work